target/linux/ramips/dts/mt7620a_edimax_br-6208ac-v2.dts

```diff
@@ -0,0 +1,257 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
```

Comment:
Any plans to upstream this device tree into the Linux kernel?

Reply:
Good idea, yes. But maybe I wait with a Linux kernel patch until my changes where reviewed here.

Reply:
I just had a look on the Linux kernel sources. They only contain a device tree for a single MT7620a device. I am afraid that the kernel maintainers are not interested in collecting DTS for small devices.

Reply:
They are. You need to give it a try. ;-) It's better when it is upstreamed, otherwise, all OpenWrt developers will take need to take care of your file and try to keep it updated. The focus could be different and they are not burdened with new things as they are already.

Reply:
While i fundamentally agree, it makes no sense to push this upstream to just apply downstream patches onto this file which are even worse to maintain.